### PR TITLE
style:added necessary home page component and structure

### PR DESCRIPTION
**Pull request by dev-enforced**
In this Pull Request one can find the structure of the Landing Page of the upcoming notes application. It has the style of the homepage which serves as a single component for now.
Here is the live preview link: https://deploy-preview-1--note-simply.netlify.app/
Didn't use my CL for making this component so some style rules might appear to be repeated which won't be seen again because I will start using it.

### CODE REVIEW

File: src/pages/HomePage/Homepage.jsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    return (
+        <section className="home-page-wrapper">
+            <div className="home-page-top">
+                <h2>
+                    <span className='home-page-title'>NOTE </span>IT
+                </h2>
+            </div>
+            <div className="home-content-container">
+                <div className="home-content">
+                    <span></span>
```
Comment (fahd23) [inline src/pages/HomePage/Homepage.jsx:15]: you can remove this span right?

**Review by fahd23 — APPROVED** [review]
Suggestion Given, Code Looks good and clean

File: src/pages/HomePage/Homepage.css [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+  top: 5%;
+  left: 5%;
+}
+
+.home-page-top{
+    color:#ff5a58;
+}
+
+.home-page-title {
+  color: #fbbc04;
```
Comment (tusharkandpal) [inline src/pages/HomePage/Homepage.css:25]: can use color variables throughout

File: src/pages/HomePage/Homepage.css [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+}
+.home-content h4 {
+  margin-left: 1rem;
+}
+.home-content-description {
+  width: 40%;
+}
+.home-content-description h1 {
+  font-size: 2rem;
+  font-family: sans-serif;
```
Comment (tusharkandpal) [inline src/pages/HomePage/Homepage.css:53]: `font-family` you can define under universal selector (*)

File: src/pages/HomePage/Homepage.jsx [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+                <Link to="/">
+                    <button className="home-page-button note">MAKE NOTES</button>
+                </Link>
+
+            </div>
+        </section>
+    )
+}
+
+export { Homepage }
```
Comment (tusharkandpal) [inline src/pages/HomePage/Homepage.jsx:41]: since you are exporting only a single component from here, you can remove `{ }` brackets

**Review by tusharkandpal — APPROVED** [review]
so far so good !

File: src/pages/HomePage/Homepage.jsx (hunk 1 above)
Comment (dev-enforced) [reply src/pages/HomePage/Homepage.jsx:15]: It is for this 
image

File: src/pages/HomePage/Homepage.css (hunk 3 above)
Comment (dev-enforced) [reply src/pages/HomePage/Homepage.css:53]: But for the other components I will use a different font family. So for the Homepage used the sans-serif font family

File: src/pages/HomePage/Homepage.jsx (hunk 4 above)
Comment (dev-enforced) [reply src/pages/HomePage/Homepage.jsx:41]: In my setup this change isn't been allowed saying it expects an expression so for that reason I am putting it in curly brackets

**Pull request merged**